opendbc/car/subaru/values.py

```diff
@@ -128,7 +122,7 @@ def init(self):
 class CAR(Platforms):
   # Global platform
   SUBARU_ASCENT = SubaruPlatformConfig(
-    [SubaruCarDocs("Subaru Ascent 2019-21", "All")],
+    [SubaruCarDocs("Subaru Ascent 2019-21", "All", footnotes=[Footnote.GLOBAL, Footnote.EXP_LONG])],
```

Comment:
Same here, you're uncoupling docs from the car interfaces. A major selling point in our docs is that things are automatic